custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -485,6 +485,16 @@ def _reg_arg_sparse_colfile(*parsers):
                 "match the number of columns in your mtx file exactly. ",
             )
 
+    @staticmethod
+    def _reg_arg_parameter_file(*parsers):
+        for parser in parsers:
+            parser.add_argument(
+                ArgumentsOptions.PARAMETER_FILE,
+                default=None,
+                type=CMRunnerArgsRegistry._is_valid_file,
+                help="Model parameters stored in a JSON file",
```

Comment:
why not call em hyperparams?

Reply:
These are the parameter values, not the hyperparam definitions

Reply:
they're hyperparameter values though.

Reply:
![Screen Shot 2021-04-09 at 10 42 27 AM](https://user-images.githubusercontent.com/10107080/114220037-53458400-9920-11eb-9aa9-40015f7917f2.png)
We call them parameters in DR so I think we should stick to that wording

Reply:
well uhh, we call them hyperparameters some places don't we? It's weird that its inconsistent.

Reply:
it shouldn't matter types vs values.

Reply:
we call the definitions hyperparameters, which define min, max, values, etc